tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_add_to_an_existing_contact_from_messages.py

```diff
@@ -46,6 +46,7 @@ def test_sms_add_number_to_existing_contact(self):
 
         contacts = Contacts(self.marionette)
         contacts.launch()
+        contacts.wait_for_contacts()
 
```

Comment:
so this assumes that contacts isn't empty right? if we're going to add it inside .launch() method, we should determine whether we're entering an empty contact screen or pre-filled one.